controllers/disruption_controller.go

```diff
@@ -652,12 +648,21 @@ func (r *DisruptionReconciler) selectTargets(instance *chaosv1beta1.Disruption)
 
 // getChaosPods returns chaos pods owned by the given instance and having the given labels
 func (r *DisruptionReconciler) getChaosPods(instance *chaosv1beta1.Disruption, ls labels.Set) ([]corev1.Pod, error) {
-	ownedPods := make([]corev1.Pod, 0)
 	pods := &corev1.PodList{}
 
-	// list pods in the instance namespace and for the given target
+	if ls == nil {
+		ls = make(map[string]string)
+	}
+
+	// If these aren't valid labels, we will instead list ALL pods
+	ls[chaostypes.DisruptionNameLabel] = instance.Name
+	ls[chaostypes.DisruptionNamespaceLabel] = instance.Namespace
+
+	r.log.Infow("Searching for chaos pods with label selector...", "labels", ls.String())
```

Comment:
```suggestion
	r.log.Infow("searching for chaos pods with label selector...", "labels", ls.String())
```
Suggesting this because we have no capital letter on other lines of logs (definitely not the most useful suggestion)